flink-runtime/src/main/java/org/apache/flink/runtime/deployment/ResultPartitionDeploymentDescriptor.java

```diff
@@ -72,6 +72,10 @@ public ResultPartitionType getPartitionType() {
 		return partitionDescriptor.getPartitionType();
 	}
 
+	public int getNumberOfPartitions() {
```

Comment:
```suggestion
	public int getTotalNumberOfAllPartitions() {
```

Reply:
just trying to leave a hint that this info goes beyond this partition (could be renamed in `PartitionDescriptor` field/getter).

Reply:
I'd be fine with renaming it to `getTotalNumberOfPartitions`; `total` and `all` in the same name is a tad confusing since it should be redundant.